### PR TITLE
added new missing functions

### PR DESCRIPTION
**Pull request by wermerb**
@istvandocsa

### CODE REVIEW

File: app/scripts/product/cart/cart.service.js [hunk 1]
```diff
@@ -26,7 +28,17 @@
       $log.debug(cart);
     }
 
-    function getCart(){
+    function removeFromCart(obj) {
+      cart = $.grep(cart, function (value) {
```
Comment (istvan-docsa-epam) [inline app/scripts/product/cart/cart.service.js]: Consider using Angular $filter

File: app/scripts/product/cart/cart.service.js [hunk 2]
```diff
@@ -26,7 +28,17 @@
       $log.debug(cart);
```
Comment (istvan-docsa-epam) [inline app/scripts/product/cart/cart.service.js]: Please add some text to the log

File: app/scripts/product/cart/cart.service.js [hunk 3]
```diff
@@ -15,6 +15,8 @@
     var cart = [];
     var service = {
       addToCart: addToCart,
+      removeFromCart: removeFromCart,
```
Comment (istvan-docsa-epam) [inline app/scripts/product/cart/cart.service.js]: maybe Cart in function names are redundant

**Comment by wermerb** [comment]
@istvan-docsa-epam @istvandocsa

File: app/scripts/product/cart/cart.service.js [hunk 4]
```diff
@@ -8,25 +8,72 @@
...
     return service;
 
     ////////////////
 
-    function addToCart(obj) {
-      cart.push(obj);
-      $log.debug(cart);
+    function add(obj) {
+      var contains = false;
+      $.each(cart, function () {
```
Comment (istvandocsa) [inline app/scripts/product/cart/cart.service.js]: angular.foreach plz

File: app/scripts/product/cart/cart.service.js [hunk 5]
```diff
@@ -8,25 +8,72 @@
...
+        if (obj.name == $(this)[0].name) {
+          $(this)[0].amount++;
+          $log.info('Increased product amount by one. Product:', $(this)[0]);
+        }
+      });
+
+    }
+
+    function removeOne(obj) {
+      $.each(cart, function () {
```
Comment (istvandocsa) [inline app/scripts/product/cart/cart.service.js]: $filter plz

**Comment by wermerb** [comment]
@istvandocsa

**Pull request merged**